oxli/khmer_api.py

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Suggest renaming this from 'khmer_api' to something else. Not sure what - maybe just 'api' or 'functions'?